core/drivers/crypto/caam/crypto.mk

```diff
@@ -153,6 +153,14 @@ CFG_CAAM_JR_DISABLE_NODE ?= y
 # 48 * 8 (Black blob overhead in bytes) = 4576 bits
 CFG_CORE_BIGNUM_MAX_BITS ?= 4576
 
+# Define if Prediction Resistance is enabled for every Random number
+# request to CAAM.
+# Performance of getting Random number from CAAM drastically decreases with
+# PR enabled.
+# Users who want CAAM RNG to get reseeded on every Random number request
+# can set this flag to y.
+CFG_CAAM_RNG_RUNTIME_PR ?= n
```

Comment:
Any particular reason for capitalizing 'Random'? How about:
```
# CAAM RNG Prediction Resistance
# When this flag is y, the CAAM RNG is reseeded on every random number request.
# In this case the performance is drastically reduced, but this makes the process
# compliant with NIST FIPS xxx.
```
(of course replace xxx with the proper reference to the FIPS requirement as requested by Jens)

Reply:
@jforissier Thanks for the comments.
Getting CAAM every time from CAAM will make the process compliant with NIST/FIPS compliant.